docs/gcp.md

```diff
@@ -0,0 +1,39 @@
+# GCP
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "gpc",
+      "project": "my-project-id",
+      "zone": "zone",
+      "domain": "domain.com",
+      "host": "@",
+      "ip_version": "ipv4"
+    }
+  ]
+}
+```
+### GOOGLE_APPLICATION_CREDENTIALS
+
+**IMPORTANT:** This provider relies on the GOOGLE_APPLICATION_CREDENTIALS environment variable to bet set.
+Make sure the variable is pointing to a valid file, which holds credentials for a Google Cloud service account.
```

Comment:
Let's just have it bind mounted to a fixed file path for now?
I don't see the point of having it configurable with an env variable, this adds complexity to the user.